deeplabcut/utils/auxfun_models.py

```diff
@@ -14,55 +14,43 @@
 from deeplabcut.utils import auxiliaryfunctions
 
 
+# This dictionary maps the model types to the file locations where the models exist.
+MODEL_BASE_PATH = Path("pose_estimation_tensorflow") / "models" / "pretrained"
+MODELTYPE_FILEPATH_MAP = {
+    "resnet_50": MODEL_BASE_PATH / "resnet_v1_50.ckpt",
+    "resnet_101": MODEL_BASE_PATH / "resnet_v1_101.ckpt",
+    "resnet_152": MODEL_BASE_PATH / "resnet_v1_152.ckpt",
+    "mobilenet_v2_1.0": MODEL_BASE_PATH / "mobilenet_v2_1.0_224.ckpt",
+    "mobilenet_v2_0.75":MODEL_BASE_PATH / "mobilenet_v2_0.75_224.ckpt",
+    "mobilenet_v2_0.5": MODEL_BASE_PATH / "mobilenet_v2_0.5_224.ckpt",
+    "mobilenet_v2_0.35":MODEL_BASE_PATH / "mobilenet_v2_0.35_224.ckpt",
+    "efficientnet-b0": MODEL_BASE_PATH / "efficientnet-b0" / "model.ckpt",
+    "efficientnet-b1": MODEL_BASE_PATH / "efficientnet-b1" / "model.ckpt",
+    "efficientnet-b2": MODEL_BASE_PATH / "efficientnet-b2" / "model.ckpt",
+    "efficientnet-b3": MODEL_BASE_PATH / "efficientnet-b3" / "model.ckpt",
+    "efficientnet-b4": MODEL_BASE_PATH / "efficientnet-b4" / "model.ckpt",
+    "efficientnet-b5": MODEL_BASE_PATH / "efficientnet-b5" / "model.ckpt",
+    "efficientnet-b6": MODEL_BASE_PATH / "efficientnet-b6" / "model.ckpt",
+}
+
+
 def Check4weights(modeltype, parent_path, num_shuffles):
     """ gets local path to network weights and checks if they are present. If not, downloads them from tensorflow.org """
-    if "resnet_50" == modeltype:
-        model_path = (
-            parent_path
-            / "pose_estimation_tensorflow/models/pretrained/resnet_v1_50.ckpt"
-        )
-    elif "resnet_101" == modeltype:
-        model_path = (
-            parent_path
-            / "pose_estimation_tensorflow/models/pretrained/resnet_v1_101.ckpt"
-        )
-    elif "resnet_152" == modeltype:
-        model_path = (
-            parent_path
-            / "pose_estimation_tensorflow/models/pretrained/resnet_v1_152.ckpt"
-        )
-    elif "mobilenet" in modeltype:
-        model_path = Path(
-            os.path.join(
-                parent_path,
-                "pose_estimation_tensorflow/models/pretrained/"
-                + str(modeltype)
-                + "_224.ckpt",
-            )
-        )
-    elif "efficientnet" in modeltype:
-        model_path = Path(
-            os.path.join(
-                parent_path,
-                "pose_estimation_tensorflow/models/pretrained/"
-                + modeltype.replace("_", "-"),
```

Comment:
Note to reviewers : This PR effectively drops support for modeltype values `efficientnet_bX` where X is [0, 6] and only supports providing modeltype as `efficientnet-bX`.

I should have pointed this out earlier @MMathisLab .

To maintain backwards compatibility, should we add the relevant `efficientnet_bX` keys to the dictionary in this PR? I didn't include them because I couldn't find any documented uses of `efficientnet_bX` which is why I ended up dropping them from the dictionary.

Reply:
I think actually wasn't used publicly (maybe for a dev repo of the effnets, thanks for the catch!)